src/actions/guides/database/pagination.cr

```diff
@@ -0,0 +1,422 @@
+class Guides::Database::Pagination < GuideAction
```

Comment:
Ah interesting. I was wondering how this would work since it affects both HTML and the Database.

Reply:
Yeah I wasn't sure what to do but decided to do it here. I was thinking of splitting it but that seems even worse. I figure people can find it with search on the site or Google.

Reply:
I think having it here is fine. We can always revisit it later if need be.